src/Storage/Bucket.php

```diff
@@ -801,8 +801,6 @@ public function isWritable($file = null)
     /**
      * Manage the IAM policy for the current Bucket.
      *
-     * Please note that this method may not yet be available in your project.
```

Comment:
@michaelbausor, you would remove the test methods in `Google\Cloud\Tests\System\ServiceWhitelist\WhitelistTest`, but storage IAM isn't covered there, so we're good to go. Thank you for catching that comment which I didn't remove before. :)